cmd/res/configuration.toml

```diff
@@ -0,0 +1,57 @@
+[Writable]
+LogLevel = 'INFO'
+
+[Service]
+BootTimeout = 30000
+CheckInterval = '10s'
+Host = 'localhost'
+ServerBindAddr = ''  # blank value defaults to Service.Host value
+Port = 49994
+Protocol = 'http'
+StartupMsg = 'device gpio started'
+Timeout = 5000
+ConnectRetries = 10
+Labels = []
+EnableAsyncReadings = false
+AsyncBufferSize = 16
+
+[Registry]
+Host = 'localhost'
+Port = 8500
+Type = 'consul'
+
+
+[Clients]
+  [Clients.Data]
+  Protocol = 'http'
+  Host = 'localhost'
+  Port = 48080
+
+  [Clients.Metadata]
+  Protocol = 'http'
+  Host = 'localhost'
+  Port = 48081
+
+[Driver]
+  Interface = 'sysfs'
+  # Interface = 'chardev'  # this interface is experimental
+  ChipSelected = '0'
```

Comment:
I think having `ChipSelected` as part of `[Driver]`, means you can only ever use one GPIO chip at a time (which will be the case most of time). It might make sense to make this the sole attribute of the `DeviceList.Protocols`? That said, as this is an experimental ABI, maybe let's leave it like this and change in 2.x if it makes sense?